packages/gpt-gateway/src/conversationType/conversationType.resolver.ts

```diff
@@ -35,8 +35,7 @@ export class ConversationTypeResolver extends ConversationTypeResolverBase {
     const conversation = await this.service.startConversionSync(args);
     return {
       success: conversation.success,
-      actionId: args.actionId,
-      requestUniqueId: conversation.requestUniqueId,
+      userActionId: args.userActionId,
```

Comment:
The GPT gateway should not know anything about user action.
it expose the requestUniqueId to allow the client to use any type of ID (can be userActionId can can be anything else)
```suggestion
      requestUniqueId: conversation.requestUniqueId,
```